lib/datasource/artifactory/__fixtures__/releases-as-folders.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<html>
+  <head>
+    <meta name="robots" content="noindex" />
+    <title>Repository Title</title>
+  </head>
+
+  <body>
+    <h1>Index</h1>
+       <pre>Name      Last modified      Size</pre><hr/>
+       <pre>
+         <a href="../">../</a>
+         <a href="1.0.0/">1.0.0/</a>  21-Jul-2021 20:08    -
```

Comment:
Why don't you simly use a regex for parsing? The benefit would allow parsing the dates as `releaseTimestamp` and using it for stabillity days.

Reply:
I think it is better to rely on a html parser instead of adding regex expression. There is a whole project that takes care of html parsing.
But, now that you mention, with the latest change, we can access the dates too from the node data. I will add that code to fill the releaseTimestamp.